modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeJavaSearchVocabulariesMethodCheck.testjava

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.source.formatter.dependencies.upgrade;
+
+/**
+ * @author Tamyris Bernardo
+ */
+public class UpgradeJavaSearchVocabulariesMethodCheck {
+
+	public void method(
+		long companyId, long groupId, String title, int start, int end) {
+
+		_assetVocabularyService.searchVocabularies(companyId, new long[]{groupId}, title, new int[]{}, start, end, new Sort());
```

Comment:
@TamyrisTorres, I just realized that this check needs to add the import associated to the class `Sort`, does it make sense to you?

Otherwise, I believe this check will introduce a compile issue.